src/lib/y2users/config_merger.rb

```diff
@@ -97,15 +101,23 @@ def find_element(config, element)
     # @param from [User, Group] Source element
     # @param to [User, Group] Target element. This element is modified.
     def copy_values(from, to)
-      return unless from.is_a?(User)
-
-      # For an user, its uid and gid should not be updated in order to keep the current AutoYaST
-      # behavior. In general, this only affects to the AutoYaST case. In other scenarios like a
-      # regular installation, all users from {rhs} (except root) are new users, so this code is
-      # not executed for them. And, during installation, only the password can be modified for root,
-      # so this actually changes nothing for the root user.
-      to.uid = from.uid
-      to.gid = from.gid
+      case from
+      when User
+        # For an user, its uid and gid should not be updated in order to keep the current AutoYaST
+        # behavior. In general, this only affects to the AutoYaST case. In other scenarios like a
+        # regular installation, all users from {rhs} (except root) are new users, so this code is
+        # not executed for them. And, during installation, only the password can be modified for root,
+        # so this actually changes nothing for the root user.
+        to.uid = from.uid
+        to.gid = from.gid
+      when Group
+        # For group it does not modify its group id as it is hard to modify it. Reason is similar as
+        # for user. Change of gid require manual steps with change of ownership of files which YaST
+        # do not want to be responsible for.
+        to.gid = from.gid
+      else
+        log.info "No specific copy for #{from.inspect}"
```

Comment:
I don't understand what this log message actually means. I would log something like: "Values can only be copied for a User or Group. Unexpected object type: #{from.inspect}."

Reply:
I agree. I was tempted to suggest a similar message than in https://github.com/yast/yast-users/blob/y2users/src/lib/y2users/config_merger.rb#L89, but then I discarded the idea.

Reply:
in such case I would turn it into error if it is not expect object.